### ISSUE TITLE
Error: Fatal trace logic error - unexpected end event on CRA

### ISSUE THREAD

**Issue by FezVrasta** [mbalabash/estimo#15]
I have a create-react-app project, I'm trying to upgrade from size-limit@1 to 2.

I installed `size-limit@2` and `@size-limit/preset-app`, which makes use of `estimo`.

My config is:
```

  "size-limit": [
    {
      "limit": "1.95MB",
      "path": "build/static/**/*.{js,css,svg,woff,woff2,png,ttf,html}"
    }
  ]
```

when I run `size-limit` I get:

```
$ size-limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 exit listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGTERM listeners added. Use emitter.setMaxListeners() to increase limit
(node:86943) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGHUP listeners added. Use emitter.setMaxListeners() to increase limit
Error: Fatal trace logic error - unexpected end event
    at Function._createTasksFromEvents (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:111:17)
    at Function.getMainThreadTasks (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:238:35)
    at Function.compute (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/lib/main-thread-tasks.js:272:28)
    at Object.computeMainThreadTasks (/Users/federicozivolo/Projects/xxx/web/node_modules/tracium/index.js:13:36)
    at generateTasksReport (/Users/federicozivolo/Projects/xxx/web/node_modules/estimo/src/reporter.js:9:25)
error Command failed with exit code 1.
```

I'm not sure what's happening, I'm on an iMac running High Sierra with 24GB of RAM

Linked to https://github.com/ai/size-limit/issues/129

**Comment by ai** [comment]
@vrizo did you have something similar?

**Comment by vrizo** [comment]
Hi @FezVrasta and @ai ,

I just added the latest `size-limit` to my simple test project created with `create-react-app`. It works perfectly on Macbook Pro with Mojave and 8 GB RAM:

```
  Size limit:   500 KB
  Size:         45.72 KB
  Loading time: 915 ms   on slow 3G
  Running time: 613 ms   on Snapdragon 410
  Total time:   1.6 s

✨  Done in 25.05s.
MacBook-Pro-Vitaly:test-uibook-with-create-react-app killbill$
```

But we had a similar error while testing our large project on Circle CI with 4 CPU and 8 GB RAM:

```
(node:646) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGINT listeners added to [process]. Use emitter.setMaxListeners() to increase limit
…
(node:646) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 SIGHUP listeners added to [process]. Use emitter.setMaxListeners() to increase limit
 ERROR  Error: EEXIST: file already exists, mkdir '/home/circleci/project/node_modules/@size-limit/.cache'
error Command failed with exit code 1.
```

Tried to rerun the tests several times but with no luck, so I had to use `@size-limit/file`.

**Comment by FezVrasta** [comment]
I think size-limit was running estimo on all the files specified in the configuration. Changing it to just `src/index.js` fixed it.

**Comment by mbalabash** [comment]
I will try to reproduce it today with CRA. 
Thanks for details @FezVrasta @vrizo

**Comment by mbalabash** [comment]
Yep, i can reproduce this issue on plain CRA project.
Bad part is it doesn't reproduce every times.

@ai 
I found some issue. Now, `size-limit` sends to estimo all type of files. Not only javascript files.

I can handle this case by filtering in `estimo`. 
But you should change [this file](https://github.com/ai/size-limit/blob/master/packages/time/get-running-time.js#L24) to:

```js
value += perf ? perf[0].javaScript / 1000 : 0
```

**Comment by ai** [comment]
I can filter files by `.js` extension

**Comment by ai** [comment]
I released filter in Size Limit 2.1.1

**Comment by mbalabash** [comment]
@vrizo @FezVrasta 
Could you try to reproduce issue with `Size Limit 2.1.1` for your case?

**Comment by mbalabash** [comment]
Seems to be fixed in `Size Limit 2.1.1`
